src/test/java/htmlverification/tests/LecPassTest.java

```diff
@@ -45,7 +45,7 @@ public void verifyExpiryDate() {
     @Test
     public void verifyVrm() {
         String vrm = certificatePageObject.getVrm();
-        assertEquals(testCertificate.getLecData().getVrm(),vrm);
+        assertEquals(String.join(" ",testCertificate.getLecData().getVrm().split("")),vrm);
```

Comment:
I can't see where the change tested here was introduced

Reply:
This is because of "blockify".  The consequence on the rendered HTML is to have a space between elements

Reply:
Ah, didn't realise that. We could use `@DisplayName("test description")` to describe tests in cases like this

Reply:
Nevermind, our Junit version is too low